bool at_ec
bool released_to_ec
bool part_all_ok
bool grasped_at_sc
int8 part_count
bool part1_exist
bool part2_exist
bool part3_exist
